static/css/settings.css

```diff
@@ -219,6 +219,7 @@
   width: 100%;
   max-width: 60rem;
   padding: 2rem;
+  padding-top: 9.6rem;
```

Comment:
I think for them to remain vertically centered this should be padding: 9.6rem 2rem;

Reply:
(i.e. also pad the bottom)

Reply:
Oh this is weird, the user-settings ul is fine but the addon-settings ul is wrong. I'm guessing this will flip the two and make the user-settings ul too padded and the addon-settings ul good.

Reply:
Ah, user-settings was also wrong, it just looked correct since it was shorter and triggered a min-height somewhere